ci/unit/required_packages.sh

```diff
@@ -30,6 +30,7 @@ pkgs="argobots                         \
       pmix                             \
       protobuf-c                       \
       spdk-devel                       \
+      strace                           \  
```

Comment:
@daltonbohning  found an extra space at the end of this line which seems to be causing the CI issues.

Reply:
Ty! Fixed.